src/traces/parcoords/plot.js

```diff
@@ -13,8 +13,9 @@ var parcoords = require('./parcoords');
 module.exports = function plot(gd, cdparcoords) {
 
     var fullLayout = gd._fullLayout;
-    var svg = fullLayout._paper;
+    var svg = fullLayout._toppaper;
```

Comment:
Why?

Reply:
For proper z-index.
`.gl-container` resides between two svg layers, unlike `.parcoord-layers` container, which was always behind svg layers.

Reply:
Right, nice. So the parcoord axes are now drawn in `_toppaper`, correct?

Reply:
@etpinard yep, I tried to test FPS - seems to work fine